draft-ietf-tls-esni.md

```diff
@@ -209,6 +208,11 @@ checksum
 of the ESNIKeys structure. For the purpose of computing the checksum, the
 value of the "checksum" field MUST be set to zero.
 
+public_name
+: The name of the entity trusted to update these encryption keys.
+This is used by the server in case there is a key mismatch or the
```

Comment:
This is used by the _client_?

Reply:
Er, I was thinking it's used by the server in that it's the identity the server assumes, but yeah conversely the client verifies it. I've reworded this to not reference either side.